### DISCUSSION TITLE
Errors loading local .bin file for GPT model

### DISCUSSION THREAD

**Discussion by ti3x** [guidance-ai/guidance#164] [Q&A]
Error when running following code:
```
guidance.llm = guidance.llms.Transformers("models/model/bin", device=0)
```

HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': 'models/model/bin_file'. Use `repo_type` argument if needed.

**Comment by slundberg** [comment]
That is the error that Transformers gives me when I pass the wrong file path.
